### PR TITLE
Update DB_Object

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: DB/common.php [hunk 1]
```diff
@@ -1161,6 +1161,16 @@ function modifyQuery($query)
         $e = new \Civi\Core\Event\QueryEvent($query);
         Civi::$statics['db_common_dispatcher']->dispatch('civi.db.query', $e);
         return $e->query;
+
```
Comment (seamuslee001) [inline DB/common.php]: @eileenmcnaughton i believe that this was an earlier version of what is in the lines above

File: DB/common.php [hunk 2]
```diff
@@ -2264,6 +2274,9 @@ function _convertNullArrayValuesToEmpty(&$array)
         }
     }
 
+    function lastInsertId() {
```
Comment (seamuslee001) [inline DB/common.php]: I believe this is a duplicate of L2288 (a modification of the patch included in #276

File: DB/common.php (hunk 1 above)
Comment (seamuslee001) [reply DB/common.php]: see https://github.com/civicrm/civicrm-packages/commit/4483923c7c066f71beae92bd3e5e7d25877538ec#diff-e87e96fe24f44eda22f348fa7adc7246

**Comment by seamuslee001** [comment]
I'm going to treat this as a reviewers cut so merging

**Comment by eileenmcnaughton** [comment]
@seamuslee001 yes I'm happy with that - it was a bit of a slog to do but we both reached the same conclusion so yay

**Pull request merged**